extensions/reviewed/AdvancedHTTP.json

```diff
@@ -0,0 +1,1329 @@
+      "fullName": "HTTP Method (Verb)",
+      "functionType": "ActionWithOperator",
+      "getterName": "RequestURL",
+      "group": "Request settings",
+      "name": "SetRequestURL",
+      "sentence": "Set HTTP method of request _PARAM1_ to _PARAM2_",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "value": "ModVarGlobalTxt"
+              },
+              "parameters": [
+                "__AdvancedHTTP.Requests[GetArgumentAsString(\"Request\")].URL",
+                "=",
+                "GetArgumentAsString(\"Value\")"
+              ]
+            }
+          ]
+        }
+      ],
+      "parameters": [
+        {
+          "description": "Request template name",
+          "name": "Request",
+          "supplementaryInformation": "sceneRequest",
+          "type": "identifier"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "the request template's target URL.",
+      "fullName": "URL",
+      "functionType": "ExpressionAndCondition",
+      "group": "Request settings",
+      "name": "RequestURL",
+      "sentence": "the URL of request template _PARAM1_",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "value": "SetReturnString"
+              },
+              "parameters": [
+                "GlobalVariableString(__AdvancedHTTP.Requests[GetArgumentAsString(\"Request\")].URL)"
+              ]
+            }
+          ]
+        }
+      ],
+      "expressionType": {
+        "supplementaryInformation": "sceneURL",
+        "type": "identifier"
+      },
+      "parameters": [
+        {
+          "description": "Request template name",
+          "name": "Request",
+          "supplementaryInformation": "sceneRequest",
+          "type": "identifier"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "CORS prevents most external websites from being queried with the browser's HTTP client, since the browser may be authenticated on that website and as such another website would be able to impersonate the player on that other website.\n\nWhen the CORS Bypass is enabled, the request will be made from a server that is not authenticated anywhere and as such is not blocked by CORS, and it eill share the response with your game. Note that as such, authentication cookies are ignored! If you own the REST API you are requesting, add CORS headers to your server instead of using this CORS Bypass.",
```

Comment:
```suggestion
      "description": "CORS prevents most external websites from being queried with the browser's HTTP client, since the browser may be authenticated on that website and as such another website would be able to impersonate the player on that other website.\n\nWhen the CORS Bypass is enabled, the request will be made from a server that is not authenticated anywhere and as such is not blocked by CORS, and it will share the response with your game. Note that as such, authentication cookies are ignored! If you own the REST API you are requesting, add CORS headers to your server instead of using this CORS Bypass.",
```